### DISCUSSION TITLE
Cross organization User search capability

### DISCUSSION THREAD

**Discussion by Jia-Li-BSY** [iTwin/community#56] [Q&A] [answered]
I am working on an iTwin project that involves collaboration between multiple organizations. I found the Users API with $search only returns metadata for users within the same organization as the caller. This seems to be a limitation for our use case, as we need to manage and collaborate with members from various organizations within the same iTwin context. Could you please confirm if there is a way to search for users from different organizations? The accesscontrol/members seems promising, but it doesn't explicitly mention support for the $search filter in its documentation.

**Comment by davidhjones** [comment]
Hi Jia Li! You are correct, the Users API only allows for you to $search amongst the users within your organization. This is by design, as we have to ensure you are only able to discover users within your organization. 

I believe you might be looking for the ability to search all members of an iTwin. Is that correct? That would mean you cannot search for ALL users, but just users which are members of an iTwin (both in your organization, but could be external participants as well).

**Reply by Jia-Li-BSY** [reply]
Correct. Does the accesscontrol/members endpoint support for the $search filter, allowing me to search for a member/user by email?

**Reply by davidhjones** [reply, marked as answer]
No, unfortunately we don’t have a way to search the members in your iTwin via API. That’s a great idea though.
